app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -1042,10 +1042,26 @@ public void onClick(View view) {
                 displayChatBox(true);
                 displayFAB(false);
                 fabHidder.start();
+                showSoftKeyboard();
+                if (streamActv.getText().toString().isEmpty() || streamActv.getText().toString().equals("")) {
```

Comment:
a better way to check if string is null or empty is [TextUtils.isEmpty()](https://developer.android.com/reference/android/text/TextUtils.html)